py-polars/polars/internals/series/series.py

```diff
@@ -5015,16 +5024,17 @@ def extend_constant(self, value: PythonLiteral | None, n: int) -> Series:
 
         """
 
-    def set_sorted(self, reverse: bool = False) -> Series:
+    @deprecate_reverse()
```

Comment:
Any reason we can't we simply use `@deprecated_alias` here?
```suggestion
    @deprecated_alias(reverse="descending")
```